index.js

```diff
@@ -35,13 +35,13 @@ const child_process = require("child_process");
 const mkdirp = require("mkdirp");
 
 /**
- * Creates a connectionConfig object given an array of ports
+ * Creates a connectionInfo object given an array of ports
  * @private
  * @param  {number[]} ports array of ports to use for the connection, [hb_port,
  *                          control_port, shell_port, stdin_port, iopub_port]
- * @return {object}         connectionConfig object
+ * @return {object}         connectionInfo object
  */
-function _createConnectionConfig(ports) {
+function _createConnectionInfo(ports) {
```

Comment:
It's ok to not have a leading underscore since we're not exporting this as a top level API.